net/rsync/files/rsyncd.json

```diff
@@ -0,0 +1,37 @@
+{
+	"bounding": [
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_SETUID",
+		"CAP_SETGID",
+		"CAP_DAC_OVERRIDE"
+	],
+	"effective": [
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_SETUID",
+		"CAP_SETGID",
+		"CAP_DAC_OVERRIDE"
+	],
+	"ambient": [
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_SETUID",
+		"CAP_SETGID",
+		"CAP_DAC_OVERRIDE"
+	],
+	"permitted": [
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_SETUID",
+		"CAP_SETGID",
+		"CAP_DAC_OVERRIDE"
+	],
+	"inheritable": [
```

Comment:
Does the inheritable set need to be the same? Am I correct that this is the set that an exec-ed process would have?

Reply:
As I understand it, the inheritable section applies to a child process. I am not sure what you mean by "need to be the same"

Reply:
I mean that if for some good or bad reason rsyncd exec's something, this something will have these capabilities, i.e. setuid, chroot, binding to lower ports, etc.
[For instance](https://download.samba.org/pub/rsync/rsyncd.conf.5#name_converter): 
> `name converter`
>    This parameter lets you specify a program that will be run by the rsync daemon to do user & group conversions between names & ids. This script is started prior to any chroot being setup, and runs as the daemon user (not the transfer user). You can specify a fully qualified pathname or a program name that is on the $PATH.

Reply:
Got it, thank you for checking this!

Reply:
you bet!